### PR TITLE
Do not prune argocd upstream resources

### PR DESCRIPTION
**Pull request by corest**

### CODE REVIEW

File: manifests/base/management-clusters-fleet-argocd.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ spec:
     namespace: argocd
   syncPolicy:
     automated:
-      prune: true 
+      prune: false
```
Comment (kopiczko) [inline manifests/base/management-clusters-fleet-argocd.yaml:35]: Why is that an issue?

Comment (corest) [reply manifests/base/management-clusters-fleet-argocd.yaml:35]: Because if you delete application, argo will delete itself

Comment (kopiczko) [reply manifests/base/management-clusters-fleet-argocd.yaml:35]: This obviously shouldn't be done, but what if they decide to delete/rename one of the resources upstream?

Comment (corest) [reply manifests/base/management-clusters-fleet-argocd.yaml:35]: dunno, prune manually ? Ok, I'll close this and we can check if that even ever happens that someone deletes this app

**Pull request closed without merging**